Makefile

```diff
@@ -0,0 +1,13 @@
+clean:
+	rm -rf dist/*
+
+dev:
+	pip install -r dev-requirements.txt
+	pip install -e .
```

Comment:
A `pip install` should always be in the same line to allow it to resolve conflicts. Please merge those 2 lines into:
`pip install -r dev-requirements.txt -e .`